editor/editor_node.cpp

```diff
@@ -4940,24 +4986,45 @@ void EditorNode::_load_docks_from_config(Ref<ConfigFile> p_layout, const String
 				atidx = k;
 				break;
 			}
-			if (atidx == -1) { // Well, it's not anywhere.
+
+			if (atidx == -1) {
+				// Try floating docks.
+				for (WindowWrapper *wrapper : floating_docks) {
+					if (wrapper->get_meta("dock_name") == name) {
```

Comment:
This doesn't work, because `wrapper->get_meta("dock_name")` is StringName and `name` is String.

Reply:
This and other fixes should be enough to make the feature finally working.